.utility/update-docs-repo.sh

```diff
@@ -13,7 +13,7 @@ git clone --quiet --branch=master git@github.com:loklak/dev.loklak.org.git lokla
 
 cd loklak_docs
 
-if ! git subtree pull --prefix=raw/server ../loklak_server documentation --squash -m "Update server subtree"
+if ! git subtree pull --prefix=raw/server ../loklak_server documentation --squash -m "Update server subtree" ; then
   git subtree add --prefix=raw/server ../loklak_server documentation --squash -m "Update server subtree"
 fi
```

Comment:
We are checking for the proper execution of `git subtree pull`, which pulls the changes of a submodule(here, docs folder of loklak_server) added in a parent repository (here, dev.loklak.org).
The only case in which this statement is supposed to fail is when the submodule hasn't been linked, which will happen only during the first deployment. For that we'll first have to add the subtree using `git subtree add`

It is kinda like, 'You cannot `git fetch upstream` if you haven't `git remote add upstream ...`'